tests/topology/input/rt-vlan-mode-link-route.yml

```diff
@@ -2,6 +2,7 @@
 # of link-, node- and vlan forwarding modes, in particular with mode = route
 #
 
+defaults.vlan.warnings.mixed_fwd: False
```

Comment:
Could scope this flag to the vlan for which we want to allow the mixed forwarding, instead of globally?

or introduce a new ```mode``` value that represents this behavior, e.g. ```mode: mixed```

Reply:
No. This an extremely niche edge case, and the only reason I didn't just shut it down is because every other blue moon someone might have a good reason to do that. I will not make a hierarchy of nerd knobs out of this ;)